site/en/tutorials/function_calling_python_quickstart.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Change the section title to Summary and merge the bullets into a paragraph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
Done.